docker/manylinux-builder/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Should this file be named `build_and_push.sh` ?

Reply:
Sure, done